index.html

```diff
@@ -211,5 +211,56 @@ <h1 class="about-heading">About me</h1>
         </li>
       </ul>
     </section>
+    <section class="contact-section">
+      <h3 class="contact-heading">
+        I'm always interested in hearing about new projects, so if you'd like to
+        chat please get in touch.
+      </h3>
+      <form
+        class="form-control"
+        action="https://formspree.io/f/mzboleyl"
+        method="post"
+      >
+        <input
+          placeholder="Full Name"
+          class="text-field"
+          type="text"
+          maxlength="30"
+          required
+        />
+        <input
+          placeholder="Email Address"
+          class="text-field"
+          type="email"
+          required
+        />
+        <textarea
+          class="message-field"
+          placeholder="Enter text here"
+          maxlength="500"
+        ></textarea>
```

Comment:
- [OPTIONAL] I suggest adding the `required` attribute in the `<textarea>` to prevent the user from sending an empty message 👌